data/operators/man_made/pipeline.json

```diff
@@ -6914,6 +6903,7 @@
       "locationSet": {
         "include": ["de-sn.geojson"]
       },
+      "matchNames": ["ensonetz"],
```

Comment:
Please check whether the matchName is correct. It seems that ensoNetz is followed by Sachsennetze, see https://www.wikidata.org/wiki/Q124370321